### PR TITLE
feat: Adds types for Chargebee.js wrappers

### PR DESCRIPTION
**Pull request by cb-farhanh**
- Adds common types for Chargebee.js wrappers.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+export interface Component {
+  name: string;
+  type: ComponentType;
+  status: Number;
+  focus: Function;
+  clear: Function;
+  blur: Function;
+  update: Function;
+  createField: Function;
+  mount(id?: any): Promise<boolean>;
```
Comment (cb-dinesh) [inline index.d.ts]: Please define a type for the argument.

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  framesCreated(): string[];
+  delegateEvent(event: CustomEvent): void;
+  tokenize(data?: any): any;
```
Comment (cb-dinesh) [inline index.d.ts]: Please define a type for the argument.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@chargebee/chargebee-js-types",
+  "version": "1.0.0",
+  "description": "TypeScript Definitions for Chargebee JS SDK",
+  "author": "",
+  "license": "MIT",
+  "scripts": {},
+  "typeScriptVersion": "4.8.4",
```
Comment (cb-dinesh) [inline package.json]: Can we include all compatible ts versions? Not all customers will be using the latest ts version.

File: index.d.ts (hunk 2 above)
Comment (cb-farhanh) [reply index.d.ts]: Done.

File: index.d.ts (hunk 1 above)
Comment (cb-farhanh) [reply index.d.ts]: Done.

File: package.json (hunk 3 above)
Comment (cb-farhanh) [reply package.json]: As discussed, [official](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-1.html#version-selection-with-typesversions) param name if `typesVersions` and not `typeScriptVersion`.
Removed incorrect one.

**Review by cb-dinesh — DISMISSED** [review]

**Review by cb-dinesh — APPROVED** [review]

**Pull request merged**